### PR TITLE
Smooth out animations and add some small player/camera features

### PR DESCRIPTION
**Pull request by Feesh7**
This adds transitions between the running / walking / jumping / hanging animations so that everything blends together nicely.

### CODE REVIEW

File: Assets/Scripts/Player/PlayerAnimator.cs [hunk 1]
```diff
@@ -108,6 +109,7 @@ public class PlayerAnimator : NetworkedBehaviour {
             }
             else {
                 isJumping = false;
+
```
Comment (aclogar) [inline Assets/Scripts/Player/PlayerAnimator.cs]: This looks like trash

Comment (Feesh7) [reply Assets/Scripts/Player/PlayerAnimator.cs]: Dammit you found it :cry:

**Comment by Feesh7** [comment]
well the gravity thing was a good idea in theory... but crouching turns you into an astronaut. I'll have to make crouching special

**Comment by nayrpg** [comment]
Is this ready for review again

**Comment by Feesh7** [comment]
hit the button whenever you want. I'm just gonna keep adding to this until it gets merged

**Pull request merged**